static/app/components/core/quote/quote.tsx

```diff
@@ -0,0 +1,41 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Stack} from '@sentry/scraps/layout';
+import {Text} from '@sentry/scraps/text';
+
+interface QuoteProps extends React.HTMLProps<HTMLElementTagNameMap['blockquote']> {
+  source?: {
+    href?: string;
+    label?: string;
+    person?: string;
+  };
+}
+export function Quote(props: QuoteProps) {
+  return (
+    <Stack as="figure" paddingLeft="xl" marginLeft="lg" borderLeft="primary">
```

Comment:
These properties will lead to `styled(Quote)`, as folks won't be able to modify them.

I would try and remove margin and padding entirely if possible, and type the props as `interface QuoteProps extends ContainerProps<'blockquote'> {source....}` with a default value for padding in cases where padding and paddingLeft props are not set - same for borderLeft.

Since it's a new component, lets add a quote.mdx with a couple examples and a index.tsx export so that it conforms to the rest of the system dir structure